README.md

```diff
@@ -9,6 +9,18 @@ For application features, build instructions and user guides see the [srsRAN doc
 
 ## Usage
 
+### Build
```

Comment:
This `build` section shouldn't be in `usage`. Most people that will use the snap won't build it.